client/my-sites/themes/auto-loading-homepage-modal.jsx

```diff
@@ -96,32 +108,72 @@ class AutoLoadingHomepageModal extends Component {
 				buttons={ [
 					{
 						action: 'keepCurrentTheme',
-						label: translate( 'No, keep my current theme' ),
+						label: postsOnFrontPage
+							? translate( 'Keep my current theme' )
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.